src/int.cr

```diff
@@ -542,6 +542,12 @@ struct Int
   # ```
   abstract def popcount
 
+  # Returns the number of leading `0`-bits.
+  # abstract def leading_zeros_count
```

Comment:
I don't think this should be left as a comment either. But please copy the doc comment to all implementations.

Reply:
Done as requested.